src/styles/variables.scss

```diff
@@ -3,4 +3,5 @@
 	--color-background: #232228;
 	--color-background-darker: #27262c;
 	--color-text: #fff;
+	--color-text-purple: #7F66EB;
```

Comment:
better to call this `--color-text-primary` this allows us to change the color later without changing the variable name

Reply:
There won't be much of the purple so let me change the white to --color-text-primary and purple to --color-text-secondary.